app/models/concerns/reports/post_edits.rb

```diff
@@ -6,6 +6,7 @@ module Reports::PostEdits
   class_methods do
     def report_post_edits(report)
       category_id, include_subcategories = report.add_category_filter
+      editor_username = report.filters.key?('editor') ? report.filters['editor'] : nil
```

Comment:
In Ruby, if the key doesn't exist, it will automatically return `nil` ;)

```suggestion
      editor_username = report.filters['editor']
```